honeybee_radiance/command/options/rtrace.py

```diff
@@ -0,0 +1,890 @@
+            'Both ti and te are set. The program can use either an include list or ' \
+            'an exclude list, but not both.'
+        assert not (self.tI.is_set and self.tE.is_set), \
+            'Both tI and tE are set. The program can use either an include list or ' \
+            'an exclude list, but not both.'
+        assert not (self.ai.is_set and self.ae.is_set), \
+            'Both ai and ae are set. The program can use either an include list or ' \
+            'an exclude list, but not both.'
+        assert not (self.aI.is_set and self.aE.is_set), \
+            'Both aI and aE are set. The program can use either an include list or ' \
+            'an exclude list, but not both.'
+        
+        if self.n.is_set and self.n > 1:
+            assert not 't' in self.o.to_radiance().lower(), \
+            'Multiple processes also do not work properly with ray tree output using' \
+            ' any of the `-o*t*` options.'
+
+        if self.n.is_set and self.x.is_set:
+            assert self.n <= self.x, \
+                'There is no benefit from specifying more processes than the -x ' \
+                'setting, which forces a wait at each flush.'
+
+        # if aa == 0 and ar is set warn user that ar value doesn't matter
+
+    @classmethod
+    def direct_studies(cls):
+        """Options for direct studies.
+
+        In particular this classmethod will set options below:
+        irradiance_calc (-I) = True
+        ambient_bounces (-ab) = 0
+        direct_certainty (-dc) = 1
+        direct_threshold (-dt) = 0
+        direct_jitter (-dj) = 0
+        direct_sec_relays (-dr) = 0
+        """
+        cls_ = cls()
+        cls_.I = True
+        cls_.ab = 0
+        cls_.dc = 1
+        cls_.dt = 0
+        cls_.dj = 0
+        cls_.dr = 0
+        return cls_
+
+    def adjust_limit_weight(self):
+        """Adjust lw to be 1 / ad if the value is larger than 1 / ad."""
+        if not self.ad.is_set:
+            print('-ad is not set.')
+            return
+        try:
+            suggested_lw = 1.0 / self.ad
+        except TypeError:
+            # ambient_divisions is not set
+            pass
+        except ZeroDivisionError:
+            # ambient_divisions is set to 0!
+            pass
+        else:
+            try:
+                lw = self.lw
+            except TypeError:
+                # lw is not set so let's set the value
+                print('-lw is set to %f.' % suggested_lw)
+                self.lw = suggested_lw
+            else:
+                if lw > suggested_lw:
+                    print('-lw is set to %f.' % suggested_lw)
+                    self.lw = suggested_lw
+                else:
+                    print('-lw ({}) is already <= {}.'.format(self.lw, suggested_lw))
+
+    @property
+    def n(self):
+        """number of rendering processes - default: 1
+        
+        Execute in parallel on nproc local processes.
+        
+        NOTE:
```

Comment:
Another all-caps NOTE.  Feel free to resolve all of these if the all-caps format is acceptable by sphinx.